src/code-examples/disclosure/css.js

```diff
@@ -1,7 +1,12 @@
-const css = `/*  These styles are optional  */
+const css = `.is-hidden {
+  display: none;
+}
```

Comment:
What do you think about refactoring Disclosure to use the `hidden` attribute? Feels more in line with the platform than using a class.